providers/utils.go

```diff
@@ -253,6 +253,8 @@ func (i *ProviderInfo) NewClient(ctx context.Context, params *NewClientParams) (
 		}
 
 		return createApiKeyHTTPClient(ctx, params.Client, params.Debug, i, params.ApiKey)
+	case Jwt:
+		fallthrough
```

Comment:
Seems like you're gonna want to implement this. All of the clients we use in the server side call this function.

Reply:
This is fine for now, since we'll special case Atlassian in the server. Can you please add a comment to explain this?